### PR TITLE
Fix: Populate parent publisher hierarchy

### PR DESCRIPTION
**Pull request by kmanaseryan**
Related issue:
https://app.zenhub.com/workspaces/datagov-multi-tenant-ckan-5cf12b601437ff1e943f0aa2/issues/gsa/datagov-ckan-multi/481

Demo:
https://www.loom.com/share/fff6bb0514694166814398b46b95c481

### CODE REVIEW

File: metadata-app/src/components/RequiredMetadata/index.js [hunk 1]
```diff
@@ -212,35 +230,27 @@ const RequiredMetadata = (props) => {
...
+            <ToolTip>
+              <h3>Publisher</h3>
+              <p>The publishing entity (e.g. your agency, department, bureau, or office).</p>
+            </ToolTip>
+          </span>
+          <Autocomplete
+            name="publisher"
+            type="string"
+            value={values.publisher}
+            // TODO - inputValue should be replaced with parent dataset name
```
Comment (kmanaseryan) [inline metadata-app/src/components/RequiredMetadata/index.js:245]: This is a leftover

Comment (anuveyatsu) [reply metadata-app/src/components/RequiredMetadata/index.js:245]: @kmanaseryan not super clear but would it still work as expected? Or is it unfinished feature?

Comment (kmanaseryan) [reply metadata-app/src/components/RequiredMetadata/index.js:245]: @anuveyatsu I meant the TODO comment is leftover, it came when I've reused the other autocomplete snippet, so don't pay attention on that line.

**Review by anuveyatsu — APPROVED** [review]
LGTM :+1:

**Pull request merged**